mixer/README.md

```diff
@@ -20,7 +45,7 @@ Mixer provides three distinct features:
   tracing and billing streams intended for the service producer itself as well
   as for its consumers.
 
-Learn more about Mixer
+Learn more about Mixer v1
```

Comment:
for some reason, my comment to remove ` v1` wasn't accepted by github.

Reply:
Your comment was on an older version. This label now says "Obsolete Mixer Documentation". Sound good?

Reply:
Obsolete Mixer Documentation SGTM, although Github continues to show me `v1` in this review. Could just be a GitHub problem - we have been noticing issues with GitHub and dif operations.